packages/libs/error-stack/src/test_helper.rs

```diff
@@ -40,7 +40,16 @@ pub fn capture_error<E>(closure: impl FnOnce() -> Result<(), Report<E>>) -> Repo
 }
 
 pub fn messages<E>(report: &Report<E>) -> Vec<String> {
-    report.frames().map(ToString::to_string).collect()
+    report
+        .frames()
+        .filter_map(|frame| match frame.kind() {
+            FrameKind::Context(context) => Some(context.to_string()),
+            FrameKind::Attachment(Attachment::Printable(attachment)) => {
+                Some(attachment.to_string())
+            }
+            FrameKind::Attachment(Attachment::Generic(_)) => None,
```

Comment:
Same comment as above on the other implementations, if we break a test we might want some more information here to help with debugging.